README.md

```diff
@@ -97,15 +97,60 @@ ARGUMENTS
   PROFILEID  Profile identifier consisting of scope (optional), profile name and its version.
 
 OPTIONS
-  -f, --force                When set to true and when profile exists in local filesystem, overwrites them.
-  -h, --help                 show CLI help
-  -i, --interactive          When set to true, command is used in interactive mode.
-  -l, --local                When set to true, profile id argument is used as a filepath to profile.supr file.
-  -p, --providers=providers  Provider name.
-  -q, --quiet                When set to true, disables the shell echo output of init actions.
-
-  -s, --scan=scan            When number provided, scan for super.json outside cwd within range represented by this
-                             number.
+  -f, --force
+      When set to true and when profile exists in local filesystem, overwrites them.
+
+  -h, --help
+      show CLI help
+
+  -i, --interactive
```

Comment:
this is way too long, i'd put it in a separate readme section and just put a condensed version here and in the command itself